antiplag/serializers.py

```diff
@@ -61,7 +61,8 @@ class Meta:
 
 class DocumentResultSerializer(serializers.ModelSerializer):
     result = ResultSerializer(read_only=True)
+    intervals = serializers.ReadOnlyField(source="result.intervals")
 
     class Meta:
         model = Document
-        fields = ("name", "result", "text")
+        fields = ("name", "result", "text", "intervals")
```

Comment:
Are intervals needed for every endpoint that uses this serializer?

Reply:
Nop, DocumentResultSerializer is used only in DocumentDetail endpoint.